orbstation/code/mob/species/tiziran/tiziran_renaming.dm

```diff
@@ -0,0 +1,91 @@
+// A variety of overrides to rename "lizardpeople" in-game.
+/obj/item/toy/plush/lizard_plushie
+	desc = "An adorable stuffed toy that resembles a tiziran."
+
+/obj/item/toy/plush/lizard_plushie/green
+	desc = "An adorable stuffed toy that resembles a green tiziran. This one fills you with nostalgia and soul."
+
+/obj/item/toy/plush/space_lizard_plushie
+	desc = "An adorable stuffed toy that resembles a very determined spacefaring tiziran. To infinity and beyond, little guy."
+
+// The original referenced "lizard janitors" which is just a baffling meme. It would be weird to just swap the name out.
+/obj/machinery/vending/wardrobe/jani_wardrobe
+	product_ads = ""
```

Comment:
does this remove the product ad completey? is that important? i know its just one wardrobe but its relevant for the sentient vending machine event or something right?